.github/PULL_REQUEST_TEMPLATE

```diff
@@ -1,32 +1,45 @@
-**Please delete this information after reading it.*
+**Problem that you are trying to solve:**
 
-Please title your PR as follows: `time: fix foo bar`. Always start with the thing you are fixing, then describe the fix.
-Don't use past tense (e.g. "fixed foo bar").
+**What you did:**
 
-Explain what your PR does and why.
+======Please delete the message past this point after reading it======
```

Comment:
Let's hope this one will get read this time :-) !

Reply:
Yeah. Let's hope.

Reply:
At the very least, the separator should now be very obvious to allow moderators to trim it for the user. (I hope this wouldn't need to happen too much)